hcx/hcxcomponents.md

```diff
@@ -0,0 +1,50 @@
+# VMware HCX Components
+
+## HCX Connector & HCX Cloud Manager
+![HCX Connector](.\images\hcx-connector.png)
+HCX Connector  
+
+![HCX Cloud Manager](.\images\hcx-cloudmanager.png)
+HCX Cloud Manager  
+
+In an HCX site-to-site architecture, there is notion of an HCX source and an HCX destination environment. This is true also for AVS-to-AVS deployments as well. Depending on the architecture and environment HCX is being deployed in, there may be a specific installer: HCX Connector or HCX Cloud. 
+
+HCX Connector is always deployed as the source. HCX Cloud is typically deployed as the destination, but it can be used as the source in AVS-to-AVS deployments. In AVS, Microsoft deploys HCX Cloud through the Add-Ons tab in the Azure Portal AVS Private Cloud blade. Customer is responsible for deploying HCX Connector on-premises.
+
+Microsoft deploys HCX Cloud in the management zone within AVS. Both HCX Cloud and HCX Connector are a one-to-one relationship to a vCenter environment.
+
+## HCX-IX - Interconnect Appliance
+
+![HCX Interconnect Appliance](.\images\hcx-ix.png)
+HCX-IX - Interconnect Appliance
+
+The HCX-IX service appliance provides replication and vMotion-based migration capabilities over VPN and ExpressRoute connections to Azure VMware Solution (AVS), providing strong encryption, traffic engineering, and virtual machine mobility.
```

Comment:
VPN and ExpressRoute and Public IP.  It is not necessary to call out the types of connectivity.  Any connectivity that meets the requirements is sufficient

Reply:
Updated